errors_mapping = {
    "30003": "IDP مارکتر را وارد کنید.",
    "30004": "موردی با IDP داده شده یافت نشد.",
    "30001": "موردی در دیتابیس یافت نشد.",
    "30002": "موردی برای ثابتهای فاکتورها یافت نشد.",
    "30072": "این ارتباط وجود دارد.",
    "30066": "کد ملی را درست وارد کنید.",
    "30007": "IDP وارد شده در دیتابیس موجود است.",
    "30006": "مارکتر در دیتابیس وجود دارد.",
    "30051": "ورودی ها را دوباره چک کنید.",
    "30050": "ورودی نام غیرقابل قبول است.",
    "30009": "IDP مارکترها را وارد کنید.",
    "30030": "IDP مارکتر و دوره را وارد کنید.",
    "30011": "مارکترها نباید یکسان باشند.",
    "30015": "کمیسیون را به درستی وارد کنید.",
    "30016": "کمیسیون را به درستی بین صفر و یک وارد کنید.",
    "30017": "تاریخ انتها را درست وارد کنید.",
    "30018": "تاریخ ابتدا را درست وارد کنید.",
    "30019": "این رابطه وجود ندارد.",
    "30071": "تاریخ پایان قبل از شروع است.",
    "30025": "PAMCode  را وارد کنید.",
    "30027": "این کاربر در تاریخهای موردنظر معامله ای نداشته است.",
    "30028": "خروجی برای متغیرهای داده شده نداریم.",
    "30010": "کمیسیون را وارد کنید.",
    "30052": "این مارکتر زیرمجموعه کسی نیست.",
    "30012": "این مارکتر زیرمجموعه نفر دیگری است.",
    "30013": "مغایرتی در تاریخ های داده شده مشاهده نشد.",
    "30090": "تاریخ اشتباه وارد شده است.",
    "30008": "موردی با متغیرهای داده شده یافت نشد.",
}


def get_error(type: str,code:str):
    if type =='TypeError':
        return {"code": code, "message": errors_mapping[code]}
    if type =='json_invalid':
        return {"code": 400, "message": code}
